tests/integration_tests.rs

```diff
@@ -179,7 +172,6 @@ async fn test_add_multi_with_a_key_that_already_exists() {
     );
 }
 
-#[ignore = "Relies on a running memcached server"]
```

Comment:
kind of nice to have a super fast test suite that doens't need memcached it has been helpful for just testing the parsers for example... we might leave these and just run two CI jobs one with and one without memcached?